### ISSUE TITLE
Support for node 4 runtime in lambda?

### ISSUE THREAD

**Issue by edclement** [serverless/serverless-runtime-babel#16]
It would be great if support for the node4 runtime in lambda could get added.
We could then switch presets from `es2015` to `es2015-node4` to reduce the amount of transpiling happening

**Comment by edclement** [comment]
Just noticed that the master branch seems to have support. Anyway we can get a tagged release?

**Comment by stramel** [comment]
https://aws.amazon.com/blogs/compute/node-js-4-3-2-runtime-now-available-on-lambda/

With es6 supported in node 4 and lambda now running 4.3.2, what of this is still valid?

**Comment by edclement** [comment]
Babel supports much more of es6 features and even some es7 that node 4/5 don't yet support. Babel will always support the latest and/or draft ecmascript spec.

**Comment by edclement** [comment]
Any progress on this? It would be nice to not have to point npm to master or a specific commit.

**Comment by minibikini** [comment]
I think it'd better to stay with the official `es2015` preset.

@edclement you could overwrite the Babel config to use the `es2015-node4`.

**Comment by aexmachina** [comment]
It would be good for this plugin to use the `nodejs4.3` runtime on Lambda because `nodejs` will be deprecated. I'll have a look at creating a PR for this now.

**Comment by edclement** [comment]
Yeah, I'm ok with that but can we get a tagged release for this commit: 

79c87b25f47122440fa0250e81837ba82e4cd533

**Comment by n8sabes** [comment]
Doesn't es2015 unnecessarily transpile a lot of logic that is already natively available in Node 4.3.x on Lambda V8 Execution Environment?

Seems that for deployed code size and speed optimization, es20515-node4 makes sense for everybody using ES6.

I'm seeing a deployment error by changing the optimizer to es2015-node4. To reproduce, create a new project with one function + endpoint (foo). Add one line to the handler, `let x = 1;` and implement serverless-optimizer but with es2015-node4. `Sls function deploy foo` ..... boom.
